src/components/EditableField.jsx

```diff
@@ -97,24 +119,37 @@ const EditableField = (props) => {
   };
 
   return (
-    <Space>
-      {renderEditState()}
-      {
-        deleteEnabled
-          ? (
-            <Tooltip placement='bottom' title='Delete' mouseLeaveDelay={0}>
-              <Button size='small' shape='circle' icon={<DeleteOutlined />} onClick={deleteEditableField} />
-            </Tooltip>
-          ) : <></>
-      }
-    </Space>
+    <>
+      <Space direction='vertical'>
+        <Space>
+          {renderEditState()}
+          {
+            deleteEnabled
+              ? (
+                <Tooltip placement='bottom' title='Delete' mouseLeaveDelay={0}>
+                  <Button size='small' shape='circle' icon={<DeleteOutlined />} onClick={deleteEditableField} />
+                </Tooltip>
+              ) : <></>
+          }
+        </Space>
+        {!isValid ? (
+          <Text type='danger'>
+            {errorText}
+            {errorFunc(editedValue)}
+          </Text>
+        ) : <></>}
+      </Space>
+    </>
   );
 };
 
 EditableField.defaultProps = {
   onAfterSubmit: () => null,
   onAfterCancel: () => null,
   onDelete: () => null,
+  validationFunc: null,
```

Comment:
if it is a func, I think a more correct way to have is as `validationFunc: () => null`

Reply:
I'll have to test this, because there's the check if `validationFunc` is set or not, and having it as `() => null` means the check will faill. Maybe it's better to set it as `validationFunc: undefined`

Reply:
`if validationFunc is set or not` <-- if `validationFunc` is a function, how can it be set? It can be either called or not, right?

Reply:
`validationFunc` should only be provided to the component if we want validation enabled, so it should be called if it is provided. Therefore, this requires a check if `validationFunc` is passed in (not undefined). Setting its default value to `validationFunc : () => null` results in `if (validationFunc)` always evaluating to true and validation always on, even if `validationFunc` is not passed in. So, I think it's default value should be `undefined` to signify that it's not passed to the component.